src/main/java/uk/gov/companieshouse/documentsigningrequestconsumer/ApiUtils.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.companieshouse.documentsigningrequestconsumer;
+
+import static java.util.Collections.singletonList;
```

Comment:
Nitpicky, but could do with a optimisation of imports ran on all the java files that have been modified.
 Just to sort out the import ordering to automatically group things like java.util imports etc.